fdm-agents/src/agents/gerrit/agent.ts

```
@@ -1,31 +1,21 @@
-        description:
-            "Expert Dutch Agronomist for fertilizer application planning.",
-        model: createDefaultModel(resolvedKey, model),
-        instruction: `You are Gerrit, an expert Dutch Agronomist.
+export const GERRIT_NAME = "Gerrit"
+export const GERRIT_DESCRIPTION =
+    "Expert Dutch Agronomist for fertilizer application planning."
+
+/** Default soft limit on tool roundtrips before the agent is warned to wrap up. */
+export const DEFAULT_TOOL_ROUND_LIMIT = 40
+
+export const TOOL_LIMIT_WARNING = `IMPORTANT: You are approaching the maximum number of allowed tool calls. You MUST produce your final JSON response NOW using the best plan you have so far. Do NOT call any more tools — output your JSON immediately.`
```

Biome (2.4.15)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)